src/openssl/hmac.cr

```diff
@@ -1,6 +1,23 @@
 require "./lib_crypto"
 
+# Allows computing Hash-based Message Authentication Code (HMAC).
+#
+# It is a type of message authentication code (MAC)
+# involving a hash function in combination with a key.
```

Comment:
This paragraph doesn't explain how to use a HMAC or what to use it for. The RFC has quite a good introduction on this topic (and is the source of truth for the implementation) so I suggest we link that: [RFC2104](https://tools.ietf.org/html/rfc2104.html).